cryptoki/tests/basic.rs

```diff
@@ -1037,7 +1037,7 @@ fn get_info_test() -> TestResult {
         assert_eq!(info.cryptoki_version().minor(), 40);
     } else {
         assert_eq!(info.cryptoki_version().major(), 3);
-        assert_eq!(info.cryptoki_version().minor(), 2);
+        assert!([0, 2].contains(&info.cryptoki_version().minor()));
```

Comment:
```suggestion
        let minor = &info.cryptoki_version().minor();
        assert!([0, 2].contains(&minor), "Only 0 and 2 are accepted but got {minor}");
```

This would make it faster to debug :)

Reply:
Good point. Updated.

Reply:
Just noticed that we would need edition 2021 for this as currently if warns as:
```
warning: panic message contains an unused formatting placeholder
    --> cryptoki/tests/basic.rs:1043:63
     |
1043 |             "Only 3.0 and 3.2 versions are expected but got 3.{minor}"
     |                                                               ^^^^^^^
     |
     = note: this message is not used as a format string when given without arguments, but will be in Rust 2021
     = note: `#[warn(non_fmt_panics)]` on by default
help: add the missing argument
     |
1043 |             "Only 3.0 and 3.2 versions are expected but got 3.{minor}", ...
     |                                                                       +++++
help: or add a "{}" format string to use the message literally
     |
1043 |             "{}", "Only 3.0 and 3.2 versions are expected but got 3.{minor}"
     |             +++++
```
Anything against changing the edition to 2021? I think the bindgen is not yet ready for 2024 edition though.